src/js/jsx/tools/SuperselectOverlay.jsx

```diff
@@ -327,7 +350,77 @@ define(function (require, exports, module) {
 
             this.updateMouseOverHighlights();
         },
+        
+        /**
+         * Draws either superselectable or leaf layer bounding boxes
+         * 
+         * @param {SVGElement} svg SVG HTML element to draw in
+         * @param {LayerTree} layerTree layerTree of the current document
+         */
+        drawMaskPreview: function (svg, layerTree, type) {
+            var scale = this._scale,
+                boundsAB = layerTree.topAncestor(layerTree.selected.first()).isArtboard ?
```

Comment:
Please assign `layerTree.topAncestor(layerTree.selected.first())` to a variable so it's not duplicated in two lines.